kmip/services/server/server.py

```diff
@@ -287,17 +287,24 @@ def interrupt_handler(trigger, frame):
         for cipher in auth_suite_ciphers:
             self._logger.debug(cipher)
 
-        self._socket = ssl.wrap_socket(
+        cafile = self.config.settings.get('ca_path')
+        context = ssl.SSLContext(self.ssl_version)
+        context.verify_mode = ssl.CERT_REQUIRED
+        if auth_suite_ciphers:
```

Comment:
This should be `self.auth_suite.ciphers`. I'm not sure exactly what the relationship between that and `auth_suite_ciphers` is, but this one crashes.

Reply:
Fixed